ratpack-core/src/main/java/ratpack/http/client/internal/InstrumentedFixedChannelPoolHandler.java

```diff
@@ -32,12 +32,6 @@ public InstrumentedFixedChannelPoolHandler(HttpChannelKey channelKey, int poolSi
     this.maxConnectionCount = poolSize;
   }
 
-  @Override
```

Comment:
Is this removed because `channelAcquired` is also called when a channel is created?

Reply:
Correct. It was only overridden to record a metric. https://github.com/netty/netty/pull/9093